### ISSUE TITLE
We need to validate that the UI and Emails are accessible to Senders and Receivers

### ISSUE THREAD

**Issue by willeybryan** [cds-snc/CANotifier#21]
**What?**
- Once we have it up and running we need to validate that the Nofity system is WCAG 2.1 Level AA compliant

**Why?**
- If we add stuff to it that isn't we will just end up making more work for ourselves. It should be so let's validate it.
